ark/jsonschema/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "@ark/jsonschema",
+	"version": "1.0.0",
+	"license": "MIT",
+	"author": {
+		"name": "TizzySaurus",
+		"email": "tizzysaurus@gmail.com",
+		"url": "https://github.com/tizzysaurus"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/arktypeio/arktype.git"
+	},
+	"type": "module",
+	"main": "./out/index.js",
+	"types": "./out/index.d.ts",
+	"exports": {
+		".": "./out/index.js",
+		"./internal/*": "./out/*"
+	},
+	"files": [
+		"out"
+	],
+	"scripts": {
+		"build": "tsx ../repo/build.ts",
```

Comment:
You'll want to update this and `exports` to match how they look in the repo now

Reply:
I think I did this correctly: b6e5c70c